app/views/layouts/_navbar.html.erb

```diff
@@ -20,8 +20,10 @@
         <li class="<%= li_active_class(root_path) %>"><%= link_if_not_active 'Dashboard', root_path %></li>
         <li class="<%= li_active_class(uploads_path) %>"><%= link_if_not_active 'Uploads', uploads_path %></li>
         <li class="<%= li_active_class(storages_path) %>"><%= link_if_not_active 'Shared Storage', storages_path %></li>
-        <li class="<%= li_active_class(crosswalk_issues_partials_path) %>"><%= link_if_not_active 'Crosswalk Partial Matches', crosswalk_issues_partials_path %></li>
-        <li class="<%= li_active_class(crosswalk_issues_orphans_path) %>"><%= link_if_not_active 'Ipeds Orphans', crosswalk_issues_orphans_path %></li>
+        <% if !production? %>
+          <li class="<%= li_active_class(crosswalk_issues_partials_path) %>"><%= link_if_not_active 'Crosswalk Partial Matches', crosswalk_issues_partials_path %></li>
+          <li class="<%= li_active_class(crosswalk_issues_orphans_path) %>"><%= link_if_not_active 'Ipeds Orphans', crosswalk_issues_orphans_path %></li>
+        <% end %>
```

Comment:
It seems like you might be better off with a specific flag `if ENV.fetch('SHOW_THIS_PARTIAL_MATCH_MENU', false)` which would let you be more selective. But if you really just want to switch off using dev/staging/prod then this obviously works